servicetalk-loadbalancer/src/main/java/io/servicetalk/loadbalancer/LoadBalancerBuilder.java

```diff
@@ -109,22 +109,6 @@ LoadBalancerBuilder<ResolvedAddress, C> healthCheckerFactory(
      */
     LoadBalancerBuilder<ResolvedAddress, C> backgroundExecutor(Executor backgroundExecutor);
 
-    /**
-     * Sets the linear search space to find an available connection for the next host.
-     * <p>
-     * When the next host has already opened connections, this {@link LoadBalancer} will perform a linear search for
-     * a connection that can serve the next request up to a specified number of attempts. If there are more open
-     * connections, selection of remaining connections will be attempted randomly.
-     * <p>
-     * Higher linear search space may help to better identify excess connections in highly concurrent environments,
-     * but may result in slightly increased selection time.
-     *
-     * @param linearSearchSpace the number of attempts for a linear search space, {@code 0} enforces random
-     * selection all the time.
-     * @return {@code this}.
-     */
-    LoadBalancerBuilder<ResolvedAddress, C> linearSearchSpace(int linearSearchSpace);
```

Comment:
I guess this technically has been in two releases, but it's unlikely someone else already started using it so its fine to remove? (vs. deprecating)

Reply:
This interface is not yet public. If we're going to, it would be good to trim this out before we do so we avoid the API compatibility questions.

Reply:
👍